app/api/chat/route.ts

```diff
@@ -0,0 +1,65 @@
+import { NextResponse } from "next/server";
+import { z, ZodError } from "zod";
+import formatZodError from "@/utils/format-zod-error";
+import { drizzle } from "drizzle-orm/vercel-postgres";
+import { sql } from "@vercel/postgres";
+import { ChatSessionsTable, InboxesTable } from "@/db/schema";
+import { eq } from "drizzle-orm";
+import { NotFoundError } from "@/server/errors";
+
+const db = drizzle(sql);
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    z
+      .object({
+        email: z.string().email(),
+        message: z.string(),
+      })
+      .parse(body);
+
+    let inbox = await db
+      .select({ id: InboxesTable.id })
+      .from(InboxesTable)
+      .where(eq(InboxesTable.name, "chat-sessions"))
```

Comment:
Let's query by InboxesTable.type for now so that we don't have a magic constant here